go/internal/test/utils.go

```diff
@@ -0,0 +1,154 @@
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/xitongsys/parquet-go-source/local"
+	"github.com/xitongsys/parquet-go/reader"
+	"github.com/xitongsys/parquet-go/source"
+)
+
+var jsonSchema string = `
+{
+	"Tag": "name=Schema, repetitiontype=REQUIRED",
+	"Fields": [
+	  {
+		"Tag": "name=Event_timestamp, type=INT64, convertedtype=TIMESTAMP_MICROS, repetitiontype=OPTIONAL"
+	  },
+	  {
+		"Tag": "name=Driver_id, type=INT64, repetitiontype=OPTIONAL"
+	  },
+	  {
+		"Tag": "name=Conv_rate, type=FLOAT, repetitiontype=OPTIONAL"
+	  },
+	  {
+		"Tag": "name=Acc_rate, type=FLOAT, repetitiontype=OPTIONAL"
+	  },
+	  {
+		"Tag": "name=Avg_daily_trips, type=INT32, repetitiontype=OPTIONAL"
+	  },
+	  {
+		"Tag": "name=Created, type=INT64, convertedtype=TIMESTAMP_MICROS, repetitiontype=OPTIONAL"
+	  }
+	]
+}
+`
+
+type Row struct {
+	Event_timestamp int64   `json:"Event_timestamp"`
+	Driver_id       int64   `json:"Driver_id"`
+	Conv_rate       float32 `json:"Conv_rate"`
+	Acc_rate        float32 `json:"Acc_rate"`
+	Avg_daily_trips int32   `json:"Avg_daily_trips"`
+	Created         int64   `json:"Created"`
+}
+
+func ReadParquet(filePath string) ([]*Row, error) {
```

Comment:
Is this method duplicated?

Reply:
Should have deleted utils.go in feast, not sure why it's not...